yarn.lock

```diff
@@ -2,27 +2,58 @@
 # yarn lockfile v1
 
 
-"@amplitude/analytics-client-common@^2.0.0-beta.0":
-  version "2.0.8"
-  resolved "https://registry.yarnpkg.com/@amplitude/analytics-client-common/-/analytics-client-common-2.0.8.tgz#6b9169afc7059e8c64ea1971e194a12ffe146e4f"
-  integrity sha512-zKD/txmMFPfSVtT2gdZw+Tf07pZQEcPcB6X39+a+Wh8PjIIADYIeq6zL/2pn/9uwMkVz66sbKABKbq69XxPfCA==
+"@amplitude/analytics-browser@^2.3.6-beta.0":
+  version "2.6.1"
+  resolved "https://registry.yarnpkg.com/@amplitude/analytics-browser/-/analytics-browser-2.6.1.tgz#b127adbe1905ce2561a73ec01d5046df3a4d5b09"
+  integrity sha512-mGHPW5gGbYJhJOskYd9t4TPpf8v5jkDB2ndCmWipSm/OXHVCcQucRGUrJdmQIRrW+5aB4ertYCK0t2m/yzj2zw==
+  dependencies:
+    "@amplitude/analytics-client-common" "^2.1.2"
+    "@amplitude/analytics-core" "^2.2.3"
+    "@amplitude/analytics-types" "^2.5.0"
+    "@amplitude/plugin-page-view-tracking-browser" "^2.2.5"
+    "@amplitude/plugin-web-attribution-browser" "^2.1.6"
+    tslib "^2.4.1"
+
+"@amplitude/analytics-client-common@^2.0.0", "@amplitude/analytics-client-common@^2.1.2":
```

Comment:
Why do we have two versions of analytics-client-common in here?

Reply:
Because we installed the analytics-client-common in multiple packages, it has a different version. 
The node and react-native are private repos, run release won't bump the version. So the versions of those packages are not synced.